### PR TITLE
Make the everything-is-a-widget principle more about views.

### PR DESCRIPTION
**Pull request by amirh**
I feel like current explanation hints that no logic should live outside of the widget tree, which IIUC is not what we want to say.

### CODE REVIEW

**Comment by sethladd** [comment]
lgtm

cc @Hixie

File: technical-overview.md [hunk 1]
```diff
@@ -55,17 +55,16 @@ principles.
 
 ### Everything's a Widget
 
-Widgets are the basic building blocks of every Flutter app. Each widget is an
+Widgets are the basic building blocks of a Flutter app's UI. Each widget is an
```
Comment (Hixie) [inline technical-overview.md]: expand abbreviations

File: technical-overview.md [hunk 2]
```diff
@@ -55,17 +55,16 @@ principles.
...
 immutable declaration of part of the user interface.  Unlike other frameworks that
-separate views, controllers, layouts, and other properties, Flutter has a
+separate views, view-controllers, layouts, and other properties, Flutter has a
```
Comment (Hixie) [inline technical-overview.md]: "view controllers" (no hyphen)

File: technical-overview.md [hunk 3]
```diff
@@ -55,17 +55,16 @@ principles.
...
 consistent, unified object model: the widget.
```
Comment (Hixie) [inline technical-overview.md]: we have controllers (separate from widgets) too. And our widgets have properties.

File: technical-overview.md [hunk 4]
```diff
@@ -55,17 +55,16 @@ principles.
...
 
 A widget can define:
 
 *   a structural element (like a button or menu)
 *   a stylistic element (like a font or color scheme)
 *   an aspect of layout (like padding)
-*   some business logic
```
Comment (Hixie) [inline technical-overview.md]: i mean, it can...

**Comment by Hixie** [comment]
LGTM modulo minor edits

**Pull request merged**